packages/venia-concept/_buildpack/create.js

```diff
@@ -105,7 +104,8 @@ function createProjectFromVenia({ fs, tasks, options }) {
                 delete config.projects.venia;
                 fs.outputJsonSync(targetPath, config, { spaces: 2 });
             },
-            '{CHANGELOG*,LICENSE*,_buildpack/*}': tasks.IGNORE,
+            '{CHANGELOG*,LICENSE*,_buildpack,_buildpack/**,**/__tests__,**/__tests__/**}':
```

Comment:
Lets extract this out into a list that's easier to manage, and then build this pattern string.

```js
const ignorePatterns = ['CHANGELOG*', 'LICENSE*', ...];
const ignoreGlob = `{${ignorePatterns.join(',')}}`; // is glob the correct term here?
```

I would also like to see a comment that these are temporarily removed until we can implement test harness into the scaffolded app.